feat(ddx): review faq ddx in scw (#3321)
scaleway/docs-content: faq/dedibox-in-scaleway-console.mdx

  h1: Dedibox in the Scaleway console
hero: assets/dedibox.webp
dates:
  validation: 2024-06-14
category: bare-metal
---

## Account linking process

### What is the account linking process for?

> The linking process is available to help you link your Dedibox (Online.net) and Scaleway accounts. Once the process is complete, you will be able to manage your Dedibox Servers and associated services from the [Scaleway console](https://console.scaleway.com/), as well as from the existing [online console](https://console.online.net).

Reviewer: ```suggestion
The linking process is available to help you link your Dedibox (Online.net) and Scaleway accounts. Once the process is complete, you will be able to manage your Dedibox servers and associated services from both the [Scaleway](https://console.scaleway.com/) and [Online](https://console.online.net) consoles.
```

  * outsourced services or if you are outsourcing your services
</Message>

### What happens during the account linking process?

> During the process, your Dedibox account is linked to your Scaleway account so that you can find all your Dedibox servers in the [new console](https://console.scaleway.com). When you do not have a Scaleway account, the process allows you to create a Scaleway account from your Dedibox information. This allows you to find all of your Dedibox servers in the new console and enjoy a new experience.

Reviewer: ```suggestion
During the process, your Dedibox account is linked to your Scaleway account so that you can find all your Dedibox servers in the [new console](https://console.scaleway.com).
If are new to Scaleway, the process allows you to create a Scaleway account from your Dedibox information. You can then find all your Dedibox servers in the Scaleway console and enjoy a new experience.
```

|                        | Failover IP                          | Order and configure your failover IP addresses. Edit your failover IP assignation.                                         | Available    |
|                        | RPN                                  | Create, list and manage all your RPN groups (V1 and V2)                                                                    | Available    |
| **BILLING**            | Payment method Management (SEPA, CB) | Manage your Scaleway payment methods (SEPA, CB) in one single place                                                        | Available    |
|                        | Invoice consultation                 | Consult your invoices for your Dedibox servers or services                                                                 | Available    |
|                        | Payment management                   | Pay for your Scaleway products with one default payment method                                                             | Available    |

### Will the new console replace the current one?

> Some operations are not yet available in the Scaleway console but you will soon be able to do everything you need. In the meantime, the Dedibox console and all the Dedibox server management features remain available after the account link.

Reviewer: ```suggestion
Some operations are not yet available in the Scaleway console, but you will soon be able to do everything you need. In the meantime, the Dedibox console and all the Dedibox server management features remain available after the account link.
```

|                        | Invoice consultation                 | Consult your invoices for your Dedibox servers or services                                                                 | Available    |
|                        | Payment management                   | Pay for your Scaleway products with one default payment method                                                             | Available    |

### Will the new console replace the current one?

Some operations are not yet available in the Scaleway console, but you will soon be able to do everything you need. In the meantime, the Dedibox console and all the Dedibox server management features remain available after the account link.

### Can I still use the Dedibox console after the upgrade?

Yes, you can continue to use the Dedibox console to manage all other Online.net services, as well as your Dedibox servers.

### When exactly will the Dedibox console (console.online.net) stop being available?

The current Dedibox console will remain accessible.

## Billing and payment information

### What impact will linking accounts have on my billing?

> You will continue to receive an invoice for each type of product (one for Scaleway Dedibox and one for Scaleway Cloud products). However, it is the payment method defined in your Scaleway console that allows you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.

Reviewer: ```suggestion
You will continue to receive separate invoices for each type of product, one for Scaleway Dedibox and one for Scaleway Cloud products. However, all invoices will be paid using the same payment method.

Therefore, once you complete the linking process, verify whether the default payment method defined in the Scaleway console is the one you want to use to pay all your future invoices (Scaleway and Dedibox).
```

### Where can I find my Dedibox invoices, following the linking of my accounts?

You will be able to find all your invoices in the Billing section of the Scaleway console (old Dedibox invoices as well as all new invoices).

### Following the linking of my accounts, will I find my Dedibox payment methods in the Scaleway console?

> All your active Dedibox payment methods will be migrated to the new console during the linking process so that you will be able to manage them from the Scaleway console instead of the Dedibox console. You have to choose a unique default payment method for your Scaleway services. This default payment method allows you to pay all of your Scaleway invoices (Scaleway Cloud and Scaleway Dedibox).

Reviewer: ```suggestion
All your active Dedibox payment methods will be migrated to the new console during the linking process so that you will be able to manage them from the Scaleway console instead of the Dedibox console. You must choose a unique default payment method for your Scaleway services, which will be used to pay all your invoices (Scaleway Cloud and Scaleway Dedibox).
```

  * If you use different credit cards for Dedibox and Scaleway Cloud, the card registered with Scaleway Cloud becomes the default payment method for both Scaleway Cloud and Scaleway Dedibox invoices.

<Message type="tip">
  Once account linking has been completed, you can modify your payment method in the [billing section](https://console.scaleway.com/billing/payment) of the Scaleway console. All existing default payment methods will be transferred to your linked account.
</Message>

### What information is synchronized between the two consoles?

We will synchronize your personal information (first name, last name, email address, phone number) and billing information (address, company name - if any, VAT number - if any, billing contact) between Scaleway and Dedibox. Once linked, you will be able to edit this information from the Scaleway console only.

<Message type="important">
  Once you have linked your account, you can edit your payment information from the Scaleway console only. You will not be able to edit the payment information from the Dedibox console anymore.
</Message>

## Future login and API

### Will I have a single login in the future?

> In the future, you may only need to use your Scaleway login information to connect to the Scaleway console and to manage all your Scaleway products.

Reviewer: ```suggestion
In the future, you may only need to use your Scaleway login information to connect to the Scaleway console and manage all your Scaleway products.
```

  * outsourced services or if you are outsourcing your services
</Message>

### What happens during the account linking process?

During the process, your Dedibox account is linked to your Scaleway account so that you can find all your Dedibox servers in the [new console](https://console.scaleway.com).
If you are new to Scaleway, the process allows you to create a Scaleway account from your Dedibox information. You can then find all your Dedibox servers in the Scaleway console and enjoy a new experience.

### What are the time constraints for linking accounts?

Accounts are linked at the user's request. You can launch the process at any time from the link provided in the email invitation.

### Is account linking free?

> Yes. The link of your Dedibox account to the Scaleway console is completely free of charge.

Reviewer 2: ```suggestion
Yes. Link your Dedibox account to the Scaleway console is completely free of charge.
```

|                        | Failover IP                          | Order and configure your failover IP addresses. Edit your failover IP assignation.                                         | Available    |
|                        | RPN                                  | Create, list and manage all your RPN groups (V1 and V2)                                                                    | Available    |
| **BILLING**            | Payment method Management (SEPA, CB) | Manage your Scaleway payment methods (SEPA, CB) in one single place                                                        | Available    |
|                        | Invoice consultation                 | Consult your invoices for your Dedibox servers or services                                                                 | Available    |
|                        | Payment management                   | Pay for your Scaleway products with one default payment method                                                             | Available    |

### Will the new console replace the current one?

> Some operations are not yet available in the Scaleway console, but you will soon be able to do everything you need. In the meantime, the Dedibox console and all the Dedibox server management features remain available after the account link.

Reviewer 2: ```suggestion
Some operations are not yet available in the Scaleway console, but you will soon be able to do everything you need. In the meantime, the Dedibox console and all the Dedibox server management features remain available after you link your account.
```
